### PR TITLE
docs: player states

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Player states docs.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/player-states.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Registering to State Changes  
+
+When a player changes its state, it fires a `PlayerStateChanged` event. The event data contains the player's old state as well as the new state. By registering to state changes and listening events, you can manage dependent logic accordingly (this applies mostly to UI changes).
+
+The state's you register to listen to include the state class, the state type, and transitions between states. These next section provide additional details.
+
+## Available State Classes  
+
+The player supports a number of state classes. You'll find a detailed list of `State` class APIs [here]().
```
Comment (OrenMe) [inline docs/player-states.md]: @dan-ziv in classes you mean CSS classes?

**Comment by AnnaAleksandrowicz** [comment]
Please LMK if updated to Oren’s changes so that I can look again.

File: docs/player-states.md (hunk 1 above)
Comment (dan-ziv) [reply docs/player-states.md]: No, my originally intent was the state class.

**Comment by dan-ziv** [comment]
@AnnaAleksandrowicz 
Can you please review the state class section?

Thanks

**Pull request merged**